### DISCUSSION TITLE
Remove statenodes.md in docs

### DISCUSSION THREAD

**Discussion by mattpocock** [statelyai/xstate#2795] [General]
The fact that there are 'States' and 'State Nodes' pages is excessive - both are so similar that one of them needs to go. 

There is a lot of content duplication between them, and having 'State nodes' as a top-level concept similar to states is confusing.

**Comment by davidkpiano** [comment]
Should state nodes be a part of the Machines page? We need to explain that a state value is comprised of selected state nodes, so the distinction is still important.
